src/main.py

```diff
@@ -349,7 +349,10 @@ def projectSingleCashflow(cf, start, end, life, lifeCf, taxMult, inflRate, proje
   projCf = np.zeros(projectLength)
   years = np.arange(projectLength) # years in project time, year 0 is first year # TODO just indices, pandas?
   # before the project starts, after it ends are zero; we want the working part
-  operatingMask = np.logical_and(years >= start, years <= end)
+  # ALFOA: Modified following expression (see issue #20):
+  #        from operatingMask = np.logical_and(years >= start, years <= end)
+  #        to operatingMask = np.logical_and(years >= start, years < end)
+  operatingMask = np.logical_and(years >= start, years < end)
```

Comment:
I do not get the point here. From the user manual, it seems to me both start and end are included in the cashflow calculations.  @PaulTalbot-INL Could you provide some comment here?

Reply:
the first year is considered the "Construction year" that is added at the initialization of the analysis (look for ```self._projectTime```)

Reply:
Does this new change capture the last year of the project time?

This is my major concern here, I see no changes to existing tests. This tells me that either our testing coverage doesn't cover replicated years, or this change has no impact on existing tests, which is also concerning.

Reply:
Yes it does. And yes, the tests were not covering this particular situation. This is the reason I added another test.

Reply:
Okay, if the last year is for sure covered, then I think this is a great PR and has my green light.

Reply:
It's surely beyond the scope of this PR, but I think we need a way to dump some of the stuff we print to screen (individual cash flows, etc) into a file so it can be reviewed; that way we could test and be sure all the correct years are being covered, the cash flows are combining correctly, and so on.

Reply:
agree. I like this...